src/test/java/com/fredsonchaves07/moviecatchapi/api/services/mail/SendEmailTest.java

```diff
@@ -18,7 +18,7 @@ public void shouldSendEmail() {
         String email = "user@email.com";
         String subject = "Teste de envio de email";
         String content = "Testando envio de email";
-        assertDoesNotThrow(() -> sendEmailService.send(new MessageEmailDTO(subject, email, content)));
+        assertDoesNotThrow(() -> sendEmailService.send(new MessageEmailDTO(subject, email, content), null));
```

Comment:
Message DTO will no longer have content as a kind of email template will be used. Strip this parameter to all methods that call it